tasks/darwin/install-package-homebrew.yml

```diff
@@ -13,14 +13,14 @@
 # This package must be manually installed for macOS <10.14, or else brew will attempt to
 # install it from a bottle and inevitably fail.
 - name: Install ccache dependencies
-  homebrew:
+  community.general.homebrew:
```

Comment:
It seems I wrote too fast... `community.general` is not listed in `requirements.yml`. Could you please address that?

Reply:
Yes, I just noticed this as well. See a118fb4.